File: P5/P73231_en/main.c
#include <stdio.h>

int max4(int a, int b, int c, int d);

int main() {
	int a, b, c, d;
	while (scanf("%d %d %d %d", &a, &b, &c, &d) > 0) {
    printf("%d\n", max4(a, b, c, d));
	}
	return 0;
}

File: P5/P73231_en/main.java
import java.util.Scanner;

class Main {

  public static void main(String[] args) {
    final Scanner sc = new Scanner(System.in);
    while (sc.hasNextInt()) {
      int a = sc.nextInt();
      int b = sc.nextInt();
      int c = sc.nextInt();
      int d = sc.nextInt();
      System.out.println(Solution.max4(a, b, c, d));
    }
  }

}
